src/site/content/en/payments/handling-optional-payment-information/index.md

```diff
@@ -200,13 +200,271 @@ object.
 Use the object to update the UI on the frontend. See [Reflect the updated
 payment details](#reflect-the-updated-payment-details).
 
+## Inform the merchant of a shipping address change {: #shipping-address-changes }
+
+Payment apps can provide the customer's shipping address to the merchant as part
+of the result of a payment transaction.
+
+This is useful for merchants because they can delegate the address collection to
+payment apps. And, because the address data will be provided in [the standard
+data format](https://w3c.github.io/payment-request/#dom-addressinit), the
+merchant can expect to receive shipping addresses in consistent structure.
+
+This is beneficial for customers as well because once they register their
+address information to their favorite payment app, they can reuse it in
+different shops.
```

Comment:
Can be rephrased as:

"Additionally, customers can register their address information with their preferred payment app and reuse it for different shops."

Should we write shops or merchants? 

I guess a question I have is—isn't this part of the main point rather than an additional benefit to the customers? They register their address, reuse it—that's part of the use case, I guess?

Reply:
Payment apps without Web Payments can already collect and provide address information, but that comes with fragmented address format. I'd put the consistent structure as more prominent benefit.